hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -778,15 +778,33 @@
       uvm_test_seq: chip_sw_alert_handler_escalation_vseq
       sw_images: ["//sw/device/tests/sim_dv:alert_handler_escalation_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
+      // Disable scoreboard to avoid incorrect alert prediction from the alert_monitor. Due to the
+      // cross-domain alert senders and receivers, the monitor from the chip level did not support
+      // processing alerts accurately from both ends.
```

Comment:
Is this something that we track in an issue to fix later (if not, should it be?)?